src/components/Events/VideoPlayer/VideoPlayer.tsx

```diff
@@ -0,0 +1,37 @@
+const useYoutubeEmbedUrl = (text: string | undefined): string | undefined => {
+  if (!text) {
+    return
+  }
+  const YOUTUBE_EMBED_BASE_URL = 'http://www.youtube.com/embed/'
+
+  const youtubeLink = text.match(
+    /(?:youtube\.com\/(?:[^\/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?\/\s]{11})/gi
+  )
+
+  return youtubeLink
+    ? YOUTUBE_EMBED_BASE_URL + youtubeLink[0].split('v=')[1]
```

Comment:
I'm afraid here you get the YT link but also the rest of the description so the iframe does not work

Reply:
Yes you're right, I was thinking that maybe we'll need the rest of the description, because It could be possible that youtube.com/embed type of links contains more than just de id.